src/loglevels.cpp

```diff
@@ -51,6 +52,11 @@ namespace g3 {
          internal::g_log_level_status[level].get().store(enabled, std::memory_order_release);
       }
 
+	  void addLogLevel(LEVELS log_level) {
+		  internal::g_default_log_level_status[log_level.value].get().store(true, std::memory_order_release);
+		  internal::g_log_level_status[log_level.value].get().store(true, std::memory_order_release);
```

Comment:
OK, so addLogLevel adds it to the default set as well.

Why not for the second paragraph call the already existing function "SetLogLevel".?